client/tests/stubs.js

```diff
@@ -1,5 +1,9 @@
+/* eslint no-restricted-globals: ["error", { "name": "jest", "message": "jest is not available in Storybook." }] */
+
 /**
- * Test stubs to mirror available global variables.
+ * Test stubs to mirror available global variables in Jest tests
+ * and Storybook, avoid using the jest global as this is not
+ * availalbe in Storybook.
```

Comment:
```suggestion
 * available in Storybook.
```

Reply:
Argh. Thanks. And yeah. Happy for this go just go in the code however works